src/routes/supplierRoutes.ts

```diff
@@ -41,39 +41,48 @@ router.get("/created-requests", async (req: Request, res: Response) => {
     "Nov",
     "Dec",
   ];
-  const tableRows = [];
-
-  for (const [key, formData] of Object.entries(acquirerForms)) {
-    key === formData.requestId;
-    let formattedDate = "Unrequested";
-    const dateValue = formData.steps.date.value as DateStep;
-
-    if (dateValue.day && dateValue.month && dateValue.year) {
-      const monthIndex = dateValue.month - 1;
-      const monthName = monthNames[monthIndex];
-      formattedDate = `${dateValue.day} ${monthName} ${dateValue.year}`;
-    }
-    const row = [
-      {
-        html: `<a href="/acquirer/${formData.dataAsset}/start">${formData.requestId}</a>`,
-      },
-      { text: formData.assetTitle },
-      { text: formData.ownedBy },
-      { text: formattedDate },
-      {
-        html: `<span class="govuk-tag ${getStatusClass(formData.status)}">${
-          formData.status
-        }</span>`,
-      },
+    
+  const allTableRows: {
+    pending: ManageShareTableRow[][],
+    submitted: ManageShareTableRow[][],
+    completed: ManageShareTableRow[][]
+  } = {
+    pending: [],
+    submitted: [[{ text: "There are no submitted data share requests.", colspan: 5 }]],
+    completed: [[{ text: "You have not completed any data share requests.", colspan: 5 }]]
+  };
+  
+  if (Object.keys(acquirerForms).length === 0) {
+    allTableRows.pending.push([{ text: "There are no pending data share requests.", colspan: 5}]);
+  } else {
+    // eslint-disable-next-line @typescript-eslint/no-unused-vars
```

Comment:
can we not do this unless its absolutely essential?
What is the `_` used for in the code below? If we leave it out does it solve the issue? (I am assuming thats the culprit causing the lint error)

Reply:
yeah I've seen that convention a lot, it made sense to me.
In this case it might be possible to use `Object.values` instead of `Object.keys` ?

Reply:
Cook, thanks @kjoshi does make sense to use values since I am not doing anything with the keys 👍🏼 

- removed the `eslint-disable`
- removed "_"
- using Object.values instead of Object.keys